app/main/routes.py

```diff
@@ -816,9 +818,9 @@ def generate_manifest(record_id: uuid.UUID):
     filename = file.FileName
     file_type = filename.split(".")[-1].lower()
 
-    if file_type == "pdf":
+    if file_type in UNIVERSAL_VIEWER_SUPPORTED_DOCUMENT_TYPES:
```

Comment:
would be good to test this `generate_manifest` route for this changed logic, i.e setup the test with different doc types, some docs, some images, an unsupported